app.js

```diff
@@ -76,12 +76,17 @@ function generateRandomImages(event) {
       button[i].removeEventListener('click', generateRandomImages);
     }
 
+    localStorage.removeItem('All Products');
+    localStorage.removeItem('Round');
+
     alert('Finished voting! View your results by clicking the results button.');
   }
   // Generate new random index 
   generateRandomIndex();
+
   // Local Storage
-  setAllProducts();
+  setItems();
+  getItems();
```

Comment:
Since we returned our parsedVersion in the function instead of setting it to a value, we need to do that here. If we don't save the return, then parsedVersion just goes away.